doc/source/admin/special_topics/interactivetools.rst

```diff
@@ -37,49 +36,272 @@ Some important benefits of using Galaxy InteractiveTools
 - **InteractiveTools** are **bonafide Galaxy Tools**; just specify **tool_type as "interactive"** and list the ports you want to expose
 - **InteractiveTools** can be **added** to and **installed from the ToolShed**.
 - **R Shiny apps**, **Javascript-based VNC** access to desktop environments, **genome-browsers-in-a-box**, **interactive notebook environments**, etc, are all possible with **InteractiveTools**
+- **InteractiveTools** typically run as software (e.g. Docker) containers in a secluded environment
```

Comment:
```suggestion
- **InteractiveTools** typically run as software (e.g. Docker) containers in an isolated environment
```

not a native speaker, but secluded sounds like they're up to no good and/or enjoying their retirement in a mountain village 😆